NDTensors/test/test_dense.jl

```diff
@@ -83,6 +83,20 @@ NDTensors.dim(i::MyInd) = i.dim
         @test A[2, 2] == Aview[1, 1]
       end
 
+      ## Testing A .= α .* B .+ β .* A
+      C = copy(A)
+      @allowscalar fill!(B, zero(elt))
+      β = elt(2.0)
+      α = elt(1.0)
+      permutedims!!(A, B, (1, 2), (a, b) -> +(*(β, a), *(α, b)))
+      @allowscalar 2.0 .* C == A
```

Comment:
```suggestion
      @allowscalar 2 .* C == A
```
to make it a bit more agnostic about the element type.